extensions/ql-vscode/src/view/variant-analysis/RepoRow.tsx

```diff
@@ -157,6 +158,8 @@ const isExpandableContentLoaded = (
   return resultsLoaded;
 };
 
+const filterRepoRowExpandedTelemetry = (v: boolean) => v === true;
```

Comment:
If `v` is a boolean, then isn't this equivalent? Otherwise, maybe the type for `v` isn't 100% correct?

```suggestion
const filterRepoRowExpandedTelemetry = (v: boolean) => v;
```

Reply:
Good point. This could definitely be simpler 👍